android/app/src/main/java/com/app/edonymyeon/presentation/ui/mypost/dialog/ConsumptionDialog.kt

```diff
@@ -0,0 +1,29 @@
+package com.app.edonymyeon.presentation.ui.mypost.dialog
+
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.DialogFragment
+import app.edonymyeon.databinding.DialogInputConsumptionBinding
+
+class ConsumptionDialog : DialogFragment() {
+
+    private lateinit var binding: DialogInputConsumptionBinding
```

Comment:
제가.. postdelete > delete dialog를 구현할 때 by lazy로 안하고 lateinit var로 구현하는 실수를 했었군요... 미안합니다... 앞으로는 컨벤션&통일성 잘 지키겠습니다.... (ㅠㅠ)

다음 코드도 다음과 같이 수정해주시면 좋을 것 같아요 ~
```suggestion
    private val binding: DialogInputConsumptionBinding by lazy {
        DialogInputConsumptionBinding.inflate(layoutInflater)
    }
```

Reply:
아아 저도 컨벤션은 생각 못한 채 사용했네요😂  이 부분은 수정하도록 하겠습니당!!